### ISSUE TITLE
Powershell 7.4.0

### ISSUE THREAD

**Issue by ThomasStubergh** [Azure/enterprise-azure-policy-as-code#433]
Hi, I updated to Powershell 7.4.0 today, and now I can't seem to run
```
Build-DeploymentPlans.ps1 -PacEnvironmentSelector tenant -OutputFolder Output
```
I just get an error message where it fail on Test-Json and it can't parse the json. I haven't made any changes on global-settings.jsonc, so I'm guessing there might be a breaking change in Powershell 7.4.0? I am attaching the screenshot with errors:
![image](https://github.com/Azure/enterprise-azure-policy-as-code/assets/38495047/5b8e883d-b20b-4d57-ac64-cb30944454b0)

I have also tried using the DefinitionsRootFolder parameter, but get the exact same error. I don't know which powershell version I was on before i did the update, but at least it was 7 something.

Hope you can shed some light on why this is happening. Let me know if you need more information.

**Comment by techlake** [comment]
Already fixed last week in latest release v8.4.6 (https://github.com/Azure/enterprise-azure-policy-as-code/releases).
Issue #426 
PR #427 

For future reference, please review latest releases and closed issues.

**Comment by ThomasStubergh** [comment]
Thanks. I searched and also looked at closed issues but didn't find the solution. Didn't think about checking latest release and changes. Will do that next time. Keep up the great work!
